tailwind.config.js

```diff
@@ -12,21 +12,37 @@ module.exports = {
         5.5: '1.375rem',
         7.5: '1.875rem',
       },
+      fontFamily: {
+        mercury: ['"PP Object Sans", sans-serif'],
+        'mercury-alt': ['"PP Monument Extended", sans-serif'],
```

Comment:
Why the icons font was deleted? It's not needed anymore?

Reply:
Thanks for noticing! It was deleted by mistake when solving conflicts.